super_team_picker/utils.js

```diff
@@ -0,0 +1,50 @@
+const utils = {
+    // Randomizing members
+    shuffle(memberArr) {
+        for (let i = 0; i < memberArr.length; i++) {
+            let random = Math.floor(Math.random() * memberArr.length);
+            [memberArr[i], memberArr[random]] = [memberArr[random], memberArr[i]]
+        }
+        return memberArr;
+    },
+    // Number of Teams for perTeam
+    numTeams(memberArr, quantity) {
+        return Math.ceil(memberArr.length / quantity);
+    },
+    // Assign teams using Per Team Method
+    perTeam(memberArr, quantity) {
+        let shuffled = this.shuffle(memberArr);
+        let numTeams = this.numTeams(memberArr, quantity);
+        let team = [];
+
+        for (let i = 1; i <= numTeams; i++) {
+            if (i !== numTeams) {
+                team.push(shuffled.slice(0, quantity));
+                shuffled.splice(0, quantity);
+            } else {
+                team.push(shuffled);
+            }
+        };
+        console.log(team)
+        return team;
```

Comment:
After you've finished debugging, removing the console.logs to prevent too many logs in the console.